### ISSUE TITLE
header option {file} is replaced by the directory name

### ISSUE THREAD

**Issue by honghaoz** [nicklockwood/SwiftFormat#445]
Looks like the header option `{file}` is still not working as expected.
For example, this option:
```
--header "{file}\nCopyright (c) ..."
```
has different output depends on how you invoke the `swiftformat` command.

- If you run the commnad by providing the file path:
  ```
  swift run swiftformat ./Sources/Foo.swift
  ```
  The output is 
  ```
  Foo.swift
  Copyright (c) ...
  ```
- However, if you run the command by providing the directory:
  ```
  swift run swiftformat ./Sources
  ```
  The output is 
  ```
  Sources
  Copyright (c) ...
  ```

The `{created}` and `{created.year}` also returns the metadata from the directory, not the file.

**Comment by DimaRU** [comment]
I already made bugfix. Please check my pull request.

**Comment by honghaoz** [comment]
#444

**Comment by honghaoz** [comment]
This is the PR: https://github.com/nicklockwood/SwiftFormat/pull/444

**Comment by nicklockwood** [comment]
@honghaoz fix merged in 0.40.10. Thanks!
